src/components/navigation/Menus/Menus.test.tsx

```diff
@@ -0,0 +1,65 @@
+import { render } from '@testing-library/react';
+import Menus from './Menus';
+
+const useRouter = jest.spyOn(require('next/router'), 'useRouter');
+useRouter.mockImplementation(() => ({
+  pathname: '/article',
+}));
+
+const menus = [
+  {
+    name: 'Article',
+    href: '/article',
+  },
+  {
+    name: 'Book',
+    href: '/book',
+  },
+  {
+    name: 'Quotes',
+    href: '/quotes',
+  },
+  {
+    name: 'About',
+    href: '/about',
+  },
+];
+
+describe('Menus', () => {
+  const renderMenus = () => render(<Menus />);
+  describe('렌더링', () => {
```

Comment:
별거 아니지만 가독성을 위한 띄어쓰기?

```suggestion
  const renderMenus = () => render(<Menus />);
  
  describe('렌더링', () => {
```

Reply:
감사합니다!